src/components/CellLineInfoCard/CellLineInfoCardBase.tsx

```diff
@@ -110,8 +108,8 @@ const CellLineInfoCard = ({
     );
     return (
         <Card title={titleContents} className={container}>
-            <InfoPanel data={tableData} />
-            <CloneTable dataSource={clones} />
+            <InfoPanel data={infoRows} />
+            {clones && <CloneTable dataSource={clones} />}
```

Comment:
this could also be an optional prop called `subTable` or `additionalInfo` that takes a component, so this component doesn't need to know about clones